src/Microsoft.Fast.Components.FluentUI/Components/NavMenu/FluentNavMenu.razor.cs

```diff
@@ -104,8 +105,13 @@ internal async Task CollapsibleClickAsync()
         }
     }
 
-    internal void HandleSelectedChange(FluentTreeItem treeItem)
+    internal async Task HandleSelectedChange(FluentTreeItem treeItem)
```

Comment:
Why do you want to make it async? If you do make it async, reflect that with an Async suffix for the method name